ai-ml/trainium-inferentia/variables.tf

```diff
@@ -1,6 +1,6 @@
 variable "name" {
   description = "Name of the VPC and EKS Cluster"
-  default     = "trainium-inferentia"
+  default     = "trainium-inferentia-sd"
```

Comment:
keep the name as `trainium-inferentia` . this is referred in many places

Reply:
Got it.